scene/gui/slider.cpp

```diff
@@ -163,6 +191,41 @@ void Slider::_update_theme_item_cache() {
 
 void Slider::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_INTERNAL_PROCESS: {
+			Input *input = Input::get_singleton();
+
+			if (input->is_action_just_released("ui_left") || input->is_action_just_released("ui_right") || input->is_action_just_released("ui_up") || input->is_action_just_released("ui_down")) {
+				gamepad_event_delay_ms = DEFAULT_GAMEPAD_EVENT_DELAY_MS;
+				set_process_internal(false);
+				return;
+			}
+
+			gamepad_event_delay_ms -= get_process_delta_time();
+			if (gamepad_event_delay_ms < 0) {
+				gamepad_event_delay_ms = gamepad_event_repeat_rate_ms;
+				if (orientation == HORIZONTAL) {
+					if (input->is_action_pressed("ui_left")) {
```

Comment:
I would probably weight the value change using the action's strength, so that leaning the thumbstick leads to a smaller step change ?

Reply:
I'm not quite sure if I understood your proposition but I think that we should not use the action strengh in this case. 

The user could find the feedback pretty strange and it could go again the industry strandard. If the user want a precise value, he just need to move the joystick one time. If he want a greater value, he can keep the joystick in the desired direction.

Reply:
Fair enough!